packages/starlight/translations/pl.json

```
@@ -0,0 +1,30 @@
+{
+  "skipLink.label": "Przejdź do głównej zawartości",
+  "search.label": "Szukaj",
+  "search.shortcutLabel": "(Naciśnij /, aby wyszukać)",
+  "search.cancelLabel": "Anuluj",
+  "search.devWarning": "Wyszukiwanie jest dostępne tylko w buildach produkcyjnych. Spróbuj zbudować i uruchomić aplikację, aby przetestować lokalnie.",
```

```suggestion
  "search.devWarning": "Wyszukiwanie jest dostępne tylko w buildach produkcyjnych. \nSpróbuj zbudować i uruchomić aplikację, aby przetestować lokalnie.",
```

The newline seems to be missing.